dpnp/dpnp_iface_mathematical.py

```diff
@@ -1394,44 +1394,50 @@ def subtract(x1, x2, **kwargs):
     return call_origin(numpy.subtract, x1, x2, **kwargs)
 
 
-def sum(x1, **kwargs):
+def sum(x1, axis=None, dtype=None, out=None, keepdims=False, initial=None, where=True):
     """
     Sum of array elements over a given axis.
 
     For full documentation refer to :obj:`numpy.sum`.
 
     Limitations
     -----------
-        Parameter ``x1`` is supported as :obj:`dpnp.ndarray`.
-        Only parameter ``axis`` from keyword arguments ``kwargs`` is supported.
-        Otherwise the functions will be executed sequentially on CPU.
-        Input array data types are limited by supported DPNP :ref:`Data types`.
+        Parameter ``x1`` is supported as :obj:`dpnp.dparray` only.
+        Parameters ``initial`` and ``where`` from keyword arguments ``kwargs`` are unsupported.
+        Input array data types are limited by DPNP :ref:`Data types`.
 
     Examples
     --------
     >>> import dpnp as np
     >>> np.sum(np.array([1, 2, 3, 4, 5]))
     15
-    >>> result = np.sum(np.array([[0, 1], [0, 5]]), axis=0)
-    >>> [x for x in result]
+    >>> result = np.sum([[0, 1], [0, 5]], axis=0)
     [0, 6]
 
     """
 
-    is_x1_dparray = isinstance(x1, dparray)
-
-    if (not use_origin_backend(x1) and is_x1_dparray):
-        axis = kwargs.get('axis')
-
-        result = dpnp_sum(x1, axis)
+    if not use_origin_backend(x1):
+        if not isinstance(x1, dparray):
+            pass
+        if out is not None and not isinstance(out, dparray):
```

Comment:
TYPO!!!!!

Reply:
@densmirn @Alexander-Makaryev It would be great to see a proposal how to avoid such typos in future. I have no idea how we could avoid it.

Reply:
it has some relation to PR #523

Reply:
> @densmirn @Alexander-Makaryev It would be great to see a proposal how to avoid such typos in future. I have no idea how we could avoid it.

@shssf tests, tests and tests :)